torch/package/_package_utils.py

```diff
@@ -0,0 +1,60 @@
+import re
+from typing import List
+
+
+def _print_file_structure(
+    filename: str, records: List[str], skip_storage: bool = True, path_includes=".*"
```

Comment:
it feels odd to have skip_storage be a very specific way to exclude storages, but path_includes be a very generic way to include files. Also in the package importer are mock and extern functions which take exclude/include pairs using file glob syntax and not regex, so using regex here feels inconsistent with that. Maybe use `glob` for both.